composer.json

```diff
@@ -26,7 +26,11 @@
         "graham-campbell/analyzer": "^2.0",
         "graham-campbell/testbench": "^5.2",
         "mockery/mockery": "^1.0",
-        "phpunit/phpunit": "^7.0"
+        "phpunit/phpunit": "^7.0",
+        "psalm/plugin-laravel": "^0.1",
+        "zendframework/zend-diactoros": "1.0",
```

Comment:
What's using this?

Reply:
It's required for Laravel IDE Helper to avoid complaints about missing packages.

Reply:
It's not actually, PR incoming...